mock/mock.spec

```diff
@@ -169,6 +169,9 @@ done
 
 ./precompile-bash-completion "mock.complete"
 
+# this is what %%sysusers_create_compat will expand to
+%{_rpmconfigdir}/sysusers.generate-pre.sh mock.conf > sysusers_script
```

Comment:
I eventually think that we do not need this `%pre` dance at all for systems with RPM >= 4.19. I'd use something like:
```
%if 0%{?fedora} && ( 0%{?rhel} <= 9 || ... others older distros may come ... )
%{_rpmconfigdir}/sysusers.generate-pre.sh mock.conf > sysusers_script
```

Reply:
Without this EL10+ condition, we risk a conflict of the "built-in" mechanism and the %pre code.